web/controllers/post_controller.ex

```diff
@@ -34,6 +34,22 @@ plug :authenticate when action in [:new, :create]
     render(conn, "index.html", posts: posts)
   end
 
+  def likes(conn, %{"id" => id}) do
```

Comment:
In an aim to be more RESTful I kinda feel like this should actually be the `#create` action in a `LikesController`? But we could keep the current URL structure the same, we should be able to do this with nested routes.